site/app/libraries/database/DatabaseQueries.php

```diff
@@ -2268,22 +2268,54 @@ public function getAverageComponentScores($g_id, $section_key, $is_team, string
         return $return;
     }
 
-    public function getAverageGraderScores($g_id, $gc_id, $section_key, $is_team) {
+    public function getAverageGraderScores($g_id, $gc_id, $section_key, $is_team, string $bad_submissions, string $null_section) {
         $u_or_t = "u";
         $users_or_teams = "users";
         $user_or_team_id = "user_id";
+        $null_section_condition = "";
+        $bad_submissions_condition = "";
+        $bad_submission_count = 0;
+        $params = [$gc_id, $g_id, $g_id, $g_id];
         if ($is_team) {
             $u_or_t = "t";
             $users_or_teams = "gradeable_teams";
             $user_or_team_id = "team_id";
         }
+        if ($null_section != 'include') {
```

Comment:
It is better to use PHP's strict comparison operators in general.

```suggestion
        if ($null_section !== 'include') {
```